dev-python/awkward/awkward-2.5.2.ebuild

```diff
@@ -0,0 +1,27 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=hatchling
+export SETUPTOOLS_SCM_PRETEND_VERSION=${PV}
+inherit distutils-r1 pypi
+
+DESCRIPTION="Manipulate JSON-like data with NumPy-like idioms."
+HOMEPAGE="https://github.com/scikit-hep/awkward"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=dev-python/numpy-1.18.0[${PYTHON_USEDEP}]
+	~dev-python/awkward-cpp-28[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-python/packaging[${PYTHON_USEDEP}]
+	dev-python/hatch-vcs[${PYTHON_USEDEP}]
+	dev-python/hatch-fancy-pypi-readme[${PYTHON_USEDEP}]
+"
+
+# disabled since it needs jax and numba
```

Comment:
Does `EPYTEST_IGNORE` and/or `EPYTEST_DESELECT` help to run a subset of tests that we do have the dependencies for?

Reply:
I have/had an ebuild for jax that I realixed was very broken, but awkward was happy with it since it only uses it's typing of arrays. I'll see if the tests are grouped in awkward.

Reply:
Ok they have ~3000 test files, I'll check which fail.